tests/test_result_to_dict.py

```diff
@@ -0,0 +1,103 @@
+import enum
+import unittest
+import datetime
+
+from sqlalchemy import Integer, Column, VARCHAR, DateTime, Enum, JSON, TEXT
+from geoalchemy2 import WKTElement
+
+from core_lib.data_transform.result_to_dict import ResultToDict, result_to_dict
+from core_lib.data_layers.data.db.sqlalchemy.base import Base
+
+
+from tests.test_data.test_utils import connect_to_mem_db
+
+
+class MyEnum(enum.Enum):
+    one = 1
+    two = 2
+    three = 3
+
+
+class Data(Base):
+    __tablename__ = 'data'
+
+    id = Column(Integer, primary_key=True, nullable=False)
+    data_name = Column(VARCHAR(length=255), nullable=False, default="")
+    data_datetime = Column(DateTime, nullable=False, default="")
+    data_text = Column(TEXT, nullable=False, default="")
+    data_json = Column(JSON, nullable=False, default="")
+    data_enum = Column(Enum(MyEnum))
+
+
+class TestResultToDict(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.db_data_session = connect_to_mem_db()
+
+    def test_lists(self):
+        lst = self.get_lists()
+        self.assertTrue(isinstance(lst, list))
+        self.assertEqual(len(lst), 3)
+        self.assertEqual(lst[0], 1)
+        self.assertEqual(lst[1], 2)
+        self.assertEqual(lst[2], 3)
+
+    def test_tuples(self):
+        tpl = self.get_tuples()
+        self.assertTrue(isinstance(tpl, tuple))
+        for t in tpl:
+            self.assertTrue(isinstance(t, tuple))

```

Comment:
thinking maybe it's also better to check the returned values.
to see that they are as expected.

maybe compare the returned value with the original value?